my-app/src/components/CommentItem.tsx

```diff
@@ -0,0 +1,26 @@
+import React from 'react';
+import styled from "styled-components";
+import {UserType} from "../pages/homePage/BlogHomeContext";
+
+type CommentType ={
+    _id?:string

```

Comment:
underscore `_` is reserved for private properties, if you use `id` and pass it in parameters why do you need to make it private? It is still not private, its just called as if it was private. 

In general I think its better to use underscore with intention, because otherwise its confusing

Reply:
fixed